package.json

```diff
@@ -125,16 +123,10 @@
         "re-resizable": "~6.5.5",
         "react": "^16.8.4",
         "react-dom": "^16.8.4",
-        "react-is": "^18.1.0",
-        "react-syntax-highlighter": "^15.5.0",
```

Comment:
None of this was required.
Looks like we were trying to force the dependencies a numbre of ways by installing manually and also using resolutions.